src/components/common/Link.tsx

```diff
@@ -50,7 +50,6 @@ export default function Link(props: LinkProps): JSX.Element {
       component='button'
       className={classNameToUse}
       onClick={onClick}
-      target={target}
```

Comment:
This seems to have been mistakenly added, this prop didn't exist in Material UI 5.11.9 even though it wasn't being reported by TS. It is failing now, and from what I can tell, we aren't passing `target` into this `Link` component anywhere within this app, so it _should_ be safe to remove.

Reply:
Good find. I think target is only relevant for the router link.